scripts/convert_pretrained_checkpoint.py

```diff
@@ -0,0 +1,83 @@
+import json
+import shutil
+import sys
+from dataclasses import asdict
+from pathlib import Path
+
+import torch
+
+# support running without installing as a package
+wd = Path(__file__).parent.parent.resolve()
+sys.path.append(str(wd))
+
+from lit_gpt import Config
+from lit_gpt.utils import incremental_save, lazy_load
+
+
+@torch.inference_mode()
+def convert_checkpoint(
+    checkpoint_file: Path,
+    tokenizer_path: Path,
+    config_name: str,
+    output_folder: Path,
+    model_key: str = "model",
```

Comment:
For simplicity, should the model key be configurable at all? The user would need a code change to change the model key saved in the pretrain/finetuning scripts. Considering that, one could argue that having it hardcoded here too is fine.